plugins/autowarmth.koplugin/main.lua

```diff
@@ -793,22 +793,23 @@ function AutoWarmth:showTimesInfo(title, location, activator, request_easy)
         face = Font:getFace("scfont"),
         width = math.floor(Screen:getWidth() * (self.easy_mode and 0.75 or 0.90)),
             text = title .. location_string .. ":\n\n" ..
-            info_line(_("Solar midnight: "), times, 1, request_easy) ..
+            info_line(_("Solar midnight:"), times, 1, request_easy) ..
             _("  Dawn\n") ..
-            info_line(_("    Astronomic: "), times, 2, request_easy) ..
-            info_line(_("    Nautical:   "), times, 3, request_easy)..
-            info_line(_("    Civil:      "), times, 4) ..
+            info_line(_("    Astronomic:"), times, 2, request_easy) ..
+            info_line(_("    Nautical:"), times, 3, request_easy)..
+            info_line(_("    Civil:"), times, 4) ..
             _("  Dawn\n") ..
-            info_line(_("Sunrise:        "), times, 5) ..
-            info_line(_("\nSolar noon:     "), times, 6, request_easy) ..
-
-            info_line(_("\nSunset:         "), times, 7) ..
+            info_line(_("Sunrise:"), times, 5) ..
+            "\n" ..
+            info_line(_("Solar noon:"), times, 6, request_easy) ..
+            "\n" ..
+            info_line(_("Sunset:"), times, 7) ..
             _("  Dusk\n") ..
-            info_line(_("    Civil:      "), times, 8) ..
-            info_line(_("    Nautical:   "), times, 9, request_easy) ..
-            info_line(_("    Astronomic: "), times, 10, request_easy) ..
+            info_line(_("    Civil:"), times, 8) ..
```

Comment:
Shouldn't the leading spaces be left out, as you did for "\n" ?

Reply:
Yep, good point.